src/patch_parse.c

```diff
@@ -109,7 +109,10 @@ static int header_path_len(git_patch_parse_ctx *ctx)
 	size_t len;
 
 	for (len = quoted; len < ctx->line_len; len++) {
-		if (!quoted && git__isspace(ctx->line[len]))
+		if (!quoted &&
+		    (!git__strncmp(ctx->line + len, " b/", strlen(" b/")) ||
+		     !git__strncmp(ctx->line + len, " \"b/", strlen(" \"b/")) ||
```

Comment:
Truly hideous, right?  Why these aren't just quoted whenever there's a space in the filename, I don't know.  😢 

Can we split the space in the filename commit out?  I _think_ this removes our ability to deal with custom prefixes.  I doubt anybody is actually using that but it might be thinking about this a bit more before we pull it out.  I think the patchid stuff is 👍 and ready to go.